test/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@
 ################################################################################
 
 list( APPEND test_input
+  testinput/amsr2_icec_l2p.nc
```

Comment:
you should have  `testinput/coperl4_icethnk.nc `instead of  `testinput/amsr2_icec_l2p.nc`